aten/src/ATen/core/interned_strings.h

```diff
@@ -0,0 +1,260 @@
+#pragma once
+#include <vector>
+#include <cstdint>
+#include <string>
+#include <unordered_map>
+#include <algorithm>
+
+#include "torch/csrc/jit/generated/aten_interned_strings.h"
```

Comment:
We talked about this: the plan is to remove this `#include` from `interned_strings.h` entirely